docs/tutorials/nextjs.mdx

```diff
@@ -0,0 +1,186 @@
+---
+id: nextjs
+title: NextJS
+sidebar_label: NextJS
+hide_title: true
+---
+
+&nbsp;
+
+# NextJS Application Tutorial
+
+:::tip What You'll Learn
+
+- How to set up and use Redux Toolkit with the NextJS framework
+
+:::
+
+:::info Prerequisites
+
+- Familiarity with [ES6 syntax and features](https://www.taniarascia.com/es6-syntax-and-feature-overview/)
+- Knowledge of React terminology: [JSX](https://reactjs.org/docs/introducing-jsx.html), [State](https://reactjs.org/docs/state-and-lifecycle.html), [Function Components, Props](https://reactjs.org/docs/components-and-props.html), and [Hooks](https://reactjs.org/docs/hooks-intro.html)
+- Understanding of [Redux terms and concepts](https://redux.js.org/tutorials/fundamentals/part-2-concepts-data-flow)
+- Working through the [Quick strt guide](./quick-start.mdx) is recommended
+
+:::
+
+## Introduction
+
+NextJS is a server side rendering framework for React that presents some unique challenges for using Redux properly. There are two architectures for a NextJS application; the Pages Router and the App Router. The Pages Router is the original architecture for NextJS. Using Redux with the Pages Router is well understood and handled primarily by the (next-redux-wrapper)[https://github.com/kirill-konshin/next-redux-wrapper]. This tutorial will focus on the App Router architecture as it is the new default architecture option for NextJS and it presents some unique challenges for using Redux properly.
```

Comment:
Early thoughts here:

- "Unique challenges" - like what?  How does this differ from a classical CRA/Vite SPA?  (maybe this gets answered later?)
- 
- "Well understood" - should probably link or point to more details here.  Might still be worth having a section somewhere in this page that talks about the challenges involved with the Pages router (multiple pages,  SSR pass,  creating multiple store instances,  passing data from server to client, etc.  There are still a lot of folks using Pages (maybe even in new projects?)

Not sure how that content should best be organized - we can iterate on that.

Conceptually, I think I'm sort of picturing a "why is any of this necessary / how is Next different than an SPA?" type section that lays out the differences, as context.